### PR TITLE
Submissions from Hacktoberfest Google Form

### PR DESCRIPTION
**Pull request by lianplass**

### CODE REVIEW

**Comment by lianplass** [comment]
Dropping in on this PR--not sure what the conflict is based on the markup.  Can you fill me in so I can correct it?

**Comment by lianplass** [comment]
@d-wasserman  see comment

**Comment by d-wasserman** [comment]
> @d-wasserman see comment

Oh boy, I will check. I bet it is the reorganizing of the TOC...

**Comment by d-wasserman** [comment]
Can you isolate the changes an re-fork the repo? There is enough changed here it is hard to figure out how to resolve all conflicts easily.

**Comment by d-wasserman** [comment]
I see ggspatial, sentinel hub, and GEEngine. Are those the only three?

**Comment by d-wasserman** [comment]
@lianplass - the conflicts here are really hard to parse quickly. It seems like we would lose some contributions if we accept this PR. Might be easier to work off a more recently clone?

**Comment by lianplass** [comment]
Hi David,

Tried to make an update from browser and think I just confused myself; will stick to VSCode next time.

-Li
